src/caretogether-pwa/src/Families/FamilyScreen.tsx

```diff
@@ -386,6 +402,23 @@ export function FamilyScreen() {
               </Grid>}
           </Grid>
           <Grid container spacing={0}>
+			<Grid item xs={12}>
+              <h3 style={{ marginBottom: 0 }}>Communities</h3>
+				{familyCommunityInfo?.map(communityInfo => {
+					return (
+						<Card variant="outlined" sx={{ paddingBottom: '1rem' }}>
+							<CardHeader 
+								sx={{ paddingBottom: 0 }}
+								title={communityInfo.community?.name}
+								subheader={communityInfo.community?.description}
```

Comment:
The description could be very long and currently is primarily used for the church address/phone/etc. Remove it for now.

Reply:
Description removed